mono/profiler/mono-profiler-log.c

```diff
@@ -5159,6 +5160,7 @@ mono_profiler_startup (const char *desc)
 		}
 		if ((opt = match_option (p, "heapshot", &val)) != p) {
 			events &= ~MONO_PROFILE_ALLOCATIONS;
+			events &= ~MONO_PROFILE_GC_MOVES;
```

Comment:
We only enable allocations if the user loads the 'Allocations' instrument.

I guess this is about what @alexrp asked yesterday ("would it be a problem for you guys if the profiler stopped producing gc move events when allocation events are disabled?")? If so, yes, if we don't enable allocations, not having GC_MOVES is ok for us. In fact, getting GC_MOVES events when we are not tracking allocations just gives us some unneeded extra work (reading those events just to find out we know nothing about those allocations being moved).